src/main/io/beeper.c

```diff
@@ -384,89 +419,85 @@ void beeperUpdate(timeUs_t currentTimeUs)
     // If beeper option from AUX switch has been selected
     if (IS_RC_MODE_ACTIVE(BOXBEEPERON)) {
         beeper(BEEPER_RX_SET);
+    }
```

Comment:
Was this a bug ?

Reply:
This is just minor refactoring, resulting code is identical, only difference is which `}` is removed by #ifdef